tests/components/switcher_kis/conftest.py

```diff
@@ -93,7 +93,7 @@ def last_state_change(self) -> datetime:
 @fixture(name="mock_bridge")
 def mock_bridge_fixture() -> Generator[None, Any, None]:
     """Fixture for mocking aioswitcher.bridge.SwitcherV2Bridge."""
-    queue = Queue()  # type: Queue
+    queue: Queue = Queue()
```

Comment:
This one too, are we really telling mypy more information it cannot derive from the value ?

Reply:
Note that we were already telling mypy that -- I didn't validate necessity of the existing type comments, but just converted them to variable annotations. But I can have a look if some of the trivial looking ones can be dropped.